test/bats/device-account.bats

```diff
@@ -69,13 +67,11 @@ random_uuid() {
 @test "device-account: upgrade" {
   token_name="$DEVICE_NAME"
   phone="$DEVICE_PHONE"
-  code="$DEVICE_CODE"
 
   variables=$(
     jq -n \
     --arg phone "$phone" \
-    --arg code "$code" \
-    '{input: {phone: $phone, code: $code}}'
+    '{input: {phone: $phone, code: "000000"}}'
```

Comment:
not sure about this, you can use a default value for the param but please keep it generic